apps/mobile/src/state/chat/utils/processIdentityRevealMessageIfAny.ts

```diff
@@ -35,13 +38,15 @@ export default function processIdentityRevealMessageIfAny(
           }
         : {
             type: 'svgXml',
-            svgXml: avatarsSvg[
-              randomNumberFromSeed(
-                0,
-                avatarsSvg.length - 1,
-                identityRevealMessage.message.deanonymizedUser.name
-              )
-            ] as SvgString,
+            svgXml:
+              chat.chat.otherSide.realLifeInfo?.anonymousImage ??
```

Comment:
Do not save the image to chat state - use the function to generate the same image for pubic key